media/css/protocol/components/_navigation-ie.scss

```diff
@@ -93,33 +75,28 @@ $image-path: '/media/protocol/img';
         margin-top: $layout-xs;
     }
 
-    .c-menu-item-title {
-        @include text-body-lg;
-    }
-
-    .c-menu-item-icon {
-        @include bidi(((float, left, right), (margin-right, $spacing-md, margin-left, $spacing-md),));
+    .c-menu-category {
+        display: inline-block;
+        *display: inline;
+        zoom: 1;
+        margin-bottom: $spacing-2xl;
     }
 
-    // hide panel close buttons
-    .c-menu-button-close {
-        display: none;
+    // only display primary navigation links to IE visitors.
```

Comment:
I made a call to only display the top-level navigation links for old IE browsers:

<img width="1677" alt="image" src="https://user-images.githubusercontent.com/400117/128155543-2254ac7a-40a7-4540-b3e7-ceeba17f82aa.png">

I figured this is probably enough, as all we can really hope for is for them to download a new browser (they are probably not going to get much out of the site itself).

Happy to try something else here if we think it's worth it.